R/estimate_R.R

```diff
@@ -139,8 +151,11 @@
 #'  \item{\code{\link{make_config}}}{ for general settings of the estimation}
 #'  \item{\code{\link{discr_si}}}{ to build serial interval distributions}
 #'  \item{\code{\link{sample_posterior_R}}}{ to draw samples of R values from
-#' the posterior distribution from the output of \code{estimate_R()}}
-#' }
+#' the posterior distribution from the output of \code{estimate_R()}
+#' }}
+#'
+#' TODO: add something about \code{dt > 1} linking to \code{\link{estimate_R_agg}}
```

Comment:
@rebeccanash please check this? can you remember what this is about?

Reply:
I think this may relate to the documentation of inter and grid, where you already have something along those lines? in that case remove?